extensions/ql-vscode/gulpfile.ts/tests.ts

```diff
@@ -6,6 +6,10 @@ export function copyTestData() {
   return Promise.resolve();
 }
 
+export function watchTestData() {
```

Comment:
And looking at `copyTestData` now, I think there is a problem. `copyNoWorkspaceData` and `copyCliIntegrationData` return promises that only resolve when the copying completes, but `copyTestData` doesn't await them. It's probably working fine now since there aren't that many files to copy, but it may break in the future if there are some big files that take a long time.

This should work:

```
export function copyTestData() {
  return Promise.all([
    copyNoWorkspaceData(),
    copyCliIntegrationData()
  ]);
}
```